src/main/java/org/folio/circulation/resources/CheckInProcessAdapter.java

```diff
@@ -170,6 +178,8 @@ CompletableFuture<Result<RequestQueue>> getRequestQueue(CheckInContext context)
   }
 
   CompletableFuture<Result<Item>> updateItem(CheckInContext context) {
+    log.debug("updateItem:: parameters context: {}", context);
```

Comment:
This logs both the loan and `CheckInByBarcodeRequest`. Do we need the full loan in the logs? Can we use Supplier or check logging level?